ui_framework/components/modal/modal.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export function KuiModal({ children, ...rest }) {
+  return (
+    <div className="kuiModal" { ...rest }>
```

Comment:
I *really* dislike this "rest" pattern. I much prefer to be explicit about the arguments you pass along, or possibly pull them off `context` or something (though I'll admit I don't even know if that's a thing, I haven't ever used `context` before...).

But, that conversation aside, doing this means that you can clobber the `className` on components. [Here's an example](https://jsfiddle.net/69z2wepo/77535/). Is this intentional? Do you want the className to be overridden?